lib/msal-browser/src/app/PublicClientApplication.ts

```diff
@@ -21,6 +21,9 @@ import { version, name } from "../packageMetadata";
  */
 export class PublicClientApplication extends ClientApplication implements IPublicClientApplication {
 
+    // Redirect Response Object
+    private silentRequest: Map<string, Promise<AuthenticationResult>>;
```

Comment:
I would propose a more detailed name, e.g. `activeSilentTokenRequests` or something like that.

Reply:
+1 since we already have a `SilentRequest` type

Reply:
Makes sense. Changed.